### PR TITLE
make this pass its own eslint, and use promise chaining because promi…

### PR DESCRIPTION
**Pull request by devsnek**
…se chaining is cool

### CODE REVIEW

**Comment by devsnek** [comment]
btw class names are **ALWAYS** capitalized.

File: spec/uploadTest.js [hunk 1]
```diff
@@ -6,7 +6,6 @@
 //  * This cannot be run inside a Node Session, use Jasmine for this tests.
 
 const owo = new (require(require('path').resolve('../owo.js/index.js')))('FAKE-TOKEN');
-const Promise = require('bluebird');
```
Comment (sr229) [inline spec/uploadTest.js]: why remove bluebird?

Comment (jakeoid) [reply spec/uploadTest.js]: BlueBirb*

Comment (sr229) [reply spec/uploadTest.js]: pls

**Pull request merged**